app/src/main/res/layout/activity_home.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#fff"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <include
+        android:id="@+id/header"
+        layout="@layout/tool_bar_main"
+        ></include>
+    <RelativeLayout
+        android:id="@+id/tabLayout"
+        android:layout_width="match_parent"
+        android:layout_height="55dp"
+        android:layout_below="@id/header">
+
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="#E3E3E4"
+            android:layout_alignParentBottom="true"/>
+
+        <ImageView
+            android:id="@+id/homeTabIv"
+            android:layout_width="45dp"
+            android:layout_height="45dp"
+            android:layout_centerVertical="true"
+            android:background="@drawable/selected_home_option_bg"
+            android:src="@drawable/home_selected_ic"
+            android:padding="9dp"
+            android:layout_marginLeft="15dp"/>
+        <ImageView
+            android:id="@+id/documentTabIv"
+            android:layout_width="45dp"
+            android:layout_height="45dp"
+            android:layout_centerVertical="true"
+            android:src="@drawable/document_non_selected_ic"
+            android:padding="9dp"
+            android:layout_toRightOf="@id/homeTabIv"
+            android:layout_marginLeft="25dp"/>
+
+        <ImageView
+            android:id="@+id/settingsIv"
+            android:layout_width="37dp"
+            android:layout_height="37dp"
+            android:layout_centerVertical="true"
+            android:src="@drawable/settings_ic"
+            android:padding="6dp"
+            android:layout_alignParentRight="true"
+            android:layout_marginRight="15dp"
+            />
+        <ImageView
+            android:id="@+id/notificationsIv"
+            android:layout_width="37dp"
+            android:layout_height="37dp"
+            android:layout_centerVertical="true"
+            android:src="@drawable/notification_ic"
+            android:padding="6dp"
+            android:layout_toLeftOf="@id/settingsIv"
+
+            />
+
+        <ImageView
+            android:id="@+id/addFileIv"
+            android:layout_width="37dp"
+            android:layout_height="37dp"
+            android:layout_centerVertical="true"
+            android:src="@drawable/add_ic"
+            android:padding="6dp"
+            android:layout_toLeftOf="@id/notificationsIv"
+
+            />
+
+    </RelativeLayout>
+
+    <FrameLayout
+        android:id="@+id/fragment_main"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@id/tabLayout">
+
+    </FrameLayout>
+
+</RelativeLayout>
```

Comment:
a nice implementation, but i would recommend using Linear Layout instead, with orientation "vertical", this will save you from adding "layout_below" for each component